policies/templates/gcp_sql_maintenance_window_v1.yaml

```diff
@@ -42,29 +42,56 @@ spec:
         # WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
```

Comment:
Looks like the UI does not let me point to the line I want to point out. 

Could you add the descriptions of your parameters on L27? Mention their type, if they are manadatory/optional, their default value (if any) and valid example values.

Reply:
Thanks for pointing this
Done